lib/arclight/custom_document.rb

```diff
@@ -37,5 +40,34 @@ def to_solr(solr_doc = {})
     def names
       [corpname, famname, name, persname].flatten.compact.uniq - repository
     end
+
+    # Combine subjets into one group from:
+    #  <controlaccess/><subject></subject>
+    #  <controlaccess/><function></function>
+    #  <controlaccess/><genreform></genreform>
+    #  <controlaccess/><occupation></occupation>
+    def access_subjects
+      subjects = search("//*[local-name()='subject' or local-name()='function' or local-name() = 'occupation' or local-name() = 'genreform']").to_a
```

Comment:
This xpath will index controlaccess sub-elements from within components into the collection (e.g. `//*`).  I have a question out to @anarchivist on #11 for clarification.